frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/SettingsUpdatesForm.unit.spec.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
 
-import MetabaseSettings from "../../../../metabase/lib/settings";
+import MetabaseSettings from "../../../../../metabase/lib/settings";
```

Comment:
Pretty sure you should be able to use an alias here and just make this `metabase/lib/settings`